api/dataprocessor.go

```diff
@@ -501,7 +501,10 @@ func (s *Server) getSeriesCachedStore(ctx *requestContext, until uint32) ([]chun
 	logLoad("cassan", ctx.AMKey, ctx.From, ctx.To)
 
 	log.Debug("cache: searching query key %s, from %d, until %d", ctx.AMKey, ctx.From, until)
-	cacheRes := s.Cache.Search(ctx.ctx, ctx.AMKey, ctx.From, until)
+	cacheRes, err := s.Cache.Search(ctx.ctx, ctx.AMKey, ctx.From, until)
+	if err != nil {
+		return iters, err
+	}
```

Comment:
this returns before the following `log.Debug()`, but i think we should either move it up so it is between the call to `.Search()` or just remove it completely if it isn't useful.